tests/integration/components/self-clear-cache-test.js

```diff
@@ -0,0 +1,124 @@
+import { module, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render, find } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+const EPOCH_TIMESTAMP = 1626242030;
+const HUMAN_TIME = ' July 13, 2021 10:53:50 PM IST';
```

Comment:
Is this a feature? Having space before the text?

Reply:
Fixed that

Reply:
Good eye @pallabez.
How does the component handle timezones though? 🤔

Reply:
In tests, we can get the time using the Date object with our provided epoch time and while implementing the component, we can do the conversion using the Date object in a similar way, It converts epoch time according to the user's current timezone

Reply:
`let myDate = new Date(1656237389*1000);`